### PR TITLE
Fix the call of the get_problem_instance service

### PR DESCRIPTION
**Pull request by teyssieuman**
Attempt to fix the problem in the test case of the `get_problem_instance` problem.

### CODE REVIEW

File: plansys2_terminal/test/terminal_test.cpp [hunk 1]
```diff
@@ -226,7 +235,12 @@ TEST(terminal_test, load_popf_plugin)
   }
 
   domain_node->trigger_transition(lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE);
-  problem_node->trigger_transition(lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE);
+
+  rclcpp_lifecycle::State state = problem_node->trigger_transition(
+    lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE);
+  std::cout << "problem_node->trigger_transition " << state.label() << std::endl;
```
Comment (fmrico) [inline plansys2_terminal/test/terminal_test.cpp:241]: I prefer to minimize output. May we change this with an ASSERT_*?

Comment (teyssieuman) [reply plansys2_terminal/test/terminal_test.cpp:241]: Ok this was debug trace that I just forgot to remove.
No problem to remove it...

Comment (teyssieuman) [inline plansys2_terminal/test/terminal_test.cpp:241]:
```suggestion
```

**Comment by fmrico** [comment]
Merging!!!

Thanks @teyssieuman

**Pull request merged**